dask/dataframe/io/hdf.py

```diff
@@ -8,6 +8,7 @@
 
 import pandas as pd
 from toolz import merge
+from ...bytes import read_bytes  # noqa
```

Comment:
May want a comment here explaining that importing `dask.bytes` has a side-effect of checking for fsspec.